doc/apimanager-reference.md

```diff
@@ -300,9 +300,10 @@ Only one of the fields can be chosen. If no field is specified then PVC is used.
 
 ### SystemS3Spec
 
-| **Field** | **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
-| --- | --- | --- | --- | --- | --- |
-| Configuration | `configurationSecretRef` | [corev1.LocalObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#localobjectreference-v1-core) | Yes | N/A | Local object reference to the secret to be used where the AWS configuration is stored. See [LocalObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#localobjectreference-v1-core) on how to specify the local object reference to the secret |
+| **Field**     | **json/yaml field**      | **Type**                                                                                                                         | **Required** | **Default value** | **Description**                                                                                                                                                                                                                                                                 |
+|---------------|--------------------------|----------------------------------------------------------------------------------------------------------------------------------|--------------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
why changed this line?

Reply:
Same as above. It was suggest to fix an error in goland that was showing.